resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -387,15 +400,24 @@ Sometimes a kernel loads, but doesn't work as expected. To test whether your ins
 
 ![](/resources_pages/imgs/jupyter_lab_r_kernel2.png)
 
+> **Note:** It might take a long time to open JupyterLab
+> when there is an R kernel notebook open
+> since the previous session.
+> Be patient and wait several seconds for JupyterLab to start the R kernel.
+
 To improve the experience of using R in JupyterLab,
 we will add keyboard shortcuts for inserting the common R operators `<-` and `|>`.
-Go to `Settings -> Advanced Settings Editor -> Keyboard Shortcuts`
-and paste the following in the rightmost panel that says `User Preferences`
-(replacing the `{}`):
+Go to `Settings -> Advanced Settings Editor -> JSON Settings Editor (top right corner) -> Keyboard Shortcuts`.
+You will see two panels,
+the right-most panel allows you to perform advanced modification
+of keyboards shortcuts in JupyterLab
+and it already contains quite a few shortcuts.
+We're going to add two more shortcuts,
+by pasting a text snippet just before the first existing shortcut.
+Go ahead and create a new line just after the line that says `"shortcuts": [`
+and paste the following:
```

Comment:
This has become more complicated due to the existing shortcuts so it is now more error prone. JupyterLab doesn't let you save if you completely screw up the JSON format, so there is no risk that a student ruins their installation, but they might delete some existing shortcuts while doing this.